templates/nimbus-vc.yaml

```diff
@@ -1,76 +1,105 @@
-            volume_mount:
-              - name: data
-                mount_path: /data/nimbus
-              - name: validators
-                mount_path: /validator_keys
-            security_context:
-              run_as_user: 0
-        container:
-          - name: vc$NODE_INDEX-nimbus
-            image: statusim/nimbus-validator-client:$NIMBUS_VERSION
-            command:
-              - sh
-              - -ac
-              - "./nimbus_validator_client \\\n --log-level=debug \\\n--data-dir=/data/nimbus \\\n--metrics \\\n--metrics-address=\"0.0.0.0\" \\\n--metrics-port=\"8008\" \\\n--beacon-node=\"http://node$NODE_INDEX:3600\" \\\n--doppelganger-detection=false \n"
-            volume_mount:
-              - name: validators
-                mount_path: /charon/node$NODE_INDEX/validator_keys
-              - name: data
-                mount_path: /data/nimbus
-        volumes:
+  selector:
+    matchLabels:
+      app: $VC_INDEX-nimbus
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        app: $VC_INDEX-nimbus
+    spec:
+      securityContext:
+        runAsUser: 0
+      initContainers:
+        - name: init-nimbus
+          image: statusim/nimbus-eth2:$NIMBUS_VERSION
+          command: [ "sh", "-c", "cp -R /home/user/nimbus_beacon_node /data/nimbus" ]
+          volumeMounts:
+            - name: data
+              mountPath: /data/nimbus
+            - name: validators
+              mountPath: /validator_keys
+          securityContext:
+            runAsUser: 0
+      containers:
+        - name: $VC_INDEX-nimbus
+          image: statusim/nimbus-validator-client:$NIMBUS_VERSION
+          command:
+            - sh
+            - -ac
+            - >
+              cp /config/import-script.sh /data/nimbus/import-script.sh;
+              chmod +x /data/nimbus/import-script.sh;
+              /data/nimbus/import-script.sh
+          volumeMounts:
+            - name: config
+              mountPath: /config
             - name: validators
-              projected:
-                sources:
-                - secret:
-                    name: node$NODE_INDEX-validators
-            name: data
-            emptyDir: {}
+              mountPath: /validator_keys
+            - name: data
+              mountPath: /data/nimbus
+      volumes:
+        - name: config
+          configMap:
+            name: import-script-config
+        - name: validators
+          projected:
+            sources:
+              - secret:
+                  name: $NODE_NAME-validators
+        - name: data
+          emptyDir: {}
 ---
 apiVersion: v1
 kind: Service
 metadata:
-  name: vc$NODE_INDEX-nimbus
-    namespace: $CLUSTER_NAME
```

Comment:
I kinda like having the vc prefix here

Reply:
yes that is assigned inside the `deploy-cluster.sh` 
`export VC_INDEX="vc$node_index"`
this was redundant, hence removed @LukeHackett12